test/testsyncfilestatustracker.cpp

```diff
@@ -148,16 +148,16 @@ private slots:
         statusSpy.clear();
         fakeFolder.execUntilItemCompleted("D");
         verifyThatPushMatchesPull(fakeFolder, statusSpy);
-        QCOMPARE(statusSpy.statusOf(""), SyncFileStatus(SyncFileStatus::StatusNone));
```

Comment:
In regards of https://github.com/owncloud/client/pull/9286 this is interesting.
I fail to interpret it however.